docker/README.md

```diff
@@ -0,0 +1,98 @@
+**This documentation focuses on the Deployer Docker image and its usage, not the Deployer itself. If you want learn more about the Deployer, please read [its documentation](../packages/deployer/README.md).**
+
+# Build
+You can build the Docker image by running:
+```bash
+docker build -f docker/Dockerfile -t api3/deployer:latest .
+```
+
+Before the build, all Deployer dependencies (but not the Deployer itself) must be available in the package registry.
+
+You can use build argument `yarnRegistry` to specify a different package registry for package installation. This is useful for testing a Docker build with unreleased dependencies. You can use for example [Verdaccio](https://verdaccio.org/) for this purpose.
```

Comment:
This one seems too technical to me and would scare me a bit :D 
Maybe mark it as "advanced usage"?

Reply:
You can use ">" syntax to make it look like a tip.